core/src/com/unciv/ui/options/DisplayTab.kt

```diff
@@ -16,14 +16,14 @@ import com.unciv.ui.utils.WrappableLabel
 import com.unciv.ui.utils.extensions.brighten
 import com.unciv.ui.utils.extensions.onChange
 import com.unciv.ui.utils.extensions.toLabel
-import com.unciv.ui.worldscreen.WorldScreen
 
 private val resolutionArray = com.badlogic.gdx.utils.Array(arrayOf("750x500", "900x600", "1050x700", "1200x800", "1500x1000"))
 
 fun displayTab(
     optionsPopup: OptionsPopup,
     onResolutionChange: () -> Unit,
-    onTilesetChange: () -> Unit
+    onTilesetChange: () -> Unit,
+    onSkinChange: () -> Unit,
```

Comment:
Since use the same function for all of these, how about replacing them with a single 'onChange' function?